__tests__/connect-spec.js

```diff
@@ -78,7 +95,7 @@ describe('Connect', () => {
               email_address: exchangeEmail,
```

Comment:
Some places we have `emailAddress` and some places `email_address`, is it possible to have this consistent throughout?

Reply:
This assertion (`toHaveBeenCalledWith`) checks the outgoing POST call. In the TS code we use `emailAddress` (camelCase) as the property name but when it gets serialized to JSON it becomes email_address (snake_case) to conform with what the API expects.